resource-definitions/template-driver/horizontalpodautoscaler/hpa-template.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: hpa-template
+entity:
+  driver_type: humanitec/template
+  name: hpa-template
+  type: horizontal-pod-autoscaler
+  driver_inputs:
+    values:
+      templates:
+        init: |
+          workload: {{ index (splitList "." "${context.res.id}") 1 }}

```

For easier reading you could define the "constants" up top here
```suggestion
          {{ $defaultMaxReplicas := 3 }}
          {{ $defaultMinReplicas := 2 }}
          {{ $absoluteMaxReplicas := 10 }}
          {{ $defaultTargetUtilizationPercent := 80 }}
          workload: {{ index (splitList "." "${context.res.id}") 1 }}

```

---

And then use these constants below.

---

Yup, good point, done.